src/Grpc.Core.Api/StatusCode.cs

```diff
@@ -120,5 +120,10 @@ public enum StatusCode
     Unavailable = 14,
 
     /// <summary>Unrecoverable data loss or corruption.</summary>
-    DataLoss = 15
+    DataLoss = 15,
+
+    /// <summary>
+    /// The operation throws an application exception.
+    /// </summary>
+    Application = 16
```

Comment:
These values come from the grpc specification. The .NET implementation needs to be compatible with other versions

Reply:
Ah ok, good to know.
Thanks so much for your time! Closing this PR then...